pkg/binfmt/Makefile

```diff
@@ -1,15 +1,4 @@
-.PHONY: tag push
-default: push
+include ../package.mk
 
-ORG?=linuxkit
 IMAGE=binfmt
-DEPS=Dockerfile Makefile main.go $(wildcard etc/binmft.d/*)
-
-HASH?=$(shell git ls-tree HEAD -- ../$(notdir $(CURDIR)) | awk '{print $$3}')
-
-tag: $(DEPS)
-	docker build --no-cache --network=none -t $(ORG)/$(IMAGE):$(HASH) .
-
-push: tag
-	DOCKER_CONTENT_TRUST=1 docker pull $(ORG)/$(IMAGE):$(HASH) || \
-	DOCKER_CONTENT_TRUST=1 docker push $(ORG)/$(IMAGE):$(HASH)
+DEPS=$(DEPS) main.go $(wildcard etc/binmft.d/*)
```

Comment:
I didn't like using `+=` as it seemed a little weird.

- `DEPS+=main.go` breaks syntax hilighting in vim and shows DEPS+ as the variable name.
- `DEPS += main.go` and `DEPS +=main.go` have the same result. This is good as you can't screw up by adding too many spaces but seems a little odd style wise. especially as the other assignments in tdon't have spaces either side of the operator.

I think I prefer the `$(BASE_DEPS) $(DEPS)` split